state/src/interface/matching_pools.rs

```diff
@@ -89,27 +89,10 @@ mod test {
 
     use crate::{matching_pools::GLOBAL_MATCHING_POOL, test_helpers::mock_state};
 
-    /// Test creating the global matching pool
-    #[tokio::test]
-    async fn test_create_global_matching_pool() {
```

Comment:
doesn't make sense to have this test anymore since the global matching pool is created in `mock_state()`